.circleci/config.yml

```diff
@@ -276,6 +285,20 @@ jobs:
           root: ~/
           paths: project/bin
 
+  cudareceiver:
+    executor:
+      name: cuda
+    steps:
+      - restore_workspace
+      - run:
+          name: Unit test for CUDA receiver
+          command: |
+            cd receiver/cudareceiver
+            make do-unit-tests-with-cover
+      - run:
+          name: Upload unit test coverage of CUDA receiver
+          command: bash <(curl -s https://codecov.io/bash) -F unit
```

Comment:
Hmm I don't have a lot of details of how this works, but does this overrides the other uploaded coverage data?

e.g. compare https://codecov.io/gh/open-telemetry/opentelemetry-collector-contrib/pull/393/tree with https://codecov.io/gh/open-telemetry/opentelemetry-collector-contrib/pull/427/tree

Reply:
Actually I wasn't sure how it works but I could not sort out how supplemental coverage addition can be achieved.

Reply:
Weirdly it seems to be working now - not sure what changed.

Reply:
@owais Could you help me with sending additional coverage info from this receiver?

Reply:
I'm not sure, but what happens if you change `-F unit` to `-F unit_cuda`? IIUC, codecov uses the flag to identify reports that are run independently so it probably shouldn't be set to the same value as the other tests.

Reply:
I tried changing the flag to `unit_cuda`